### PR TITLE
feat: Replace public files api with platform in expense card lite component

### PR DESCRIPTION
**Pull request by Chethan-Fyle**
### Description
copilot:summary

copilot:poem

### Walkthrough
copilot:walkthrough

## Clickup
https://app.clickup.com/t/86cvbvr7d

## Code Coverage
Please add code coverage here

## UI Preview
Please add screenshots for UI changes

### CODE REVIEW

File: src/app/core/models/matched-expense.model.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export interface MatchedExpense {
```
Comment (abhishek1234321) [inline src/app/core/models/matched-expense.model.ts:1]: If everything is optional why not use a Partial?

Comment (abhishek1234321) [reply src/app/core/models/matched-expense.model.ts:1]: and why is everything optional? What is the point of the model then?

**Review by abhishek1234321 — CHANGES_REQUESTED** [review]

File: src/app/core/models/matched-expense.model.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  id: string;
+  currency: string;
+  amount: number;
+  spent_at: string;
```
Comment (rvab) [inline src/app/core/models/matched-expense.model.ts:5]: Shouldn't the type be a `Date`?

File: src/app/core/models/matched-expense.model.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  txn_dt: string;
```
Comment (rvab) [inline src/app/core/models/matched-expense.model.ts:6]: same as above

File: src/app/core/models/matched-expense.model.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  merchant: string;
+  vendor: string;
+  foreign_currency: string;
+  foreign_amount: number;
+  purpose: string;
+  state: string;
+  seq_num: string;
+  no_of_files: number;
+  category_display_name: string;
```
Comment (rvab) [inline src/app/core/models/matched-expense.model.ts:15]: is the key `category_display_name`  added on the FE side?

File: src/app/core/models/matched-expense.model.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  currency: string;
+  amount: number;
+  spent_at: string;
+  txn_dt: string;
+  merchant: string;
+  vendor: string;
+  foreign_currency: string;
+  foreign_amount: number;
+  purpose: string;
+  state: string;
```
Comment (rvab) [inline src/app/core/models/matched-expense.model.ts:12]: Shouldn't this be an enum?

**Review by rvab — APPROVED** [review]
Few comments

**Review by abhishek1234321 — APPROVED** [review]

**Pull request merged**